### ISSUE TITLE
[Bug]: "@User" comments shown as "/channel/ID"

### ISSUE THREAD

**Issue by Winterhuman** [FreeTubeApp/FreeTube#2232]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.

### Describe the bug

1. Go to  "Europe's toughest airport landing used to be a lot harder" (`6kolTgj7uQc`) and compare to https://yewtu.be/watch?v=6kolTgj7uQc.
2. Look at the replies under "Philip Brechler".
3. See that yewtu.be correctly shows "@ThePmso", but, FreeTube shows `/channel/UCl_bPpK-OdmlxSK_lhjItSQ` in it's place.

### Expected Behavior

`/channel/ID` should be `@User` like how Invidious instances display it.

### Issue Labels

text/string issue

### FreeTube Version

v0.16.0 Beta

### Operating System Version

Arch Linux, 5.17.5-arch1-1, x86_64

### Installation Method (If applicable)

Flathub

### Last Known Working FreeTube Version (If Any)

v0.16.0; might be a change from YouTube's side?

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
both showing normal for me?

**Comment by Winterhuman** [comment]
Are you sure? Do you have a screenshot comparing @ThePmso's reply under **Philip Brechler**'s comment (one from FreeTube using the Local API and one from https://yewtu.be the website)

**Comment by Winterhuman** [comment]
Oh wait, now it's fixed? It's showing "@ThePmso" now. Perhaps it's more sporadic?

**Comment by Winterhuman** [comment]
Ah, it only shows as `/channel/ID` when Invidious API is the default, that's why it showed before; the local API failed when I saw the issue.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
So just to be clear i have to set invidous api as default api and then do both searches?

**Comment by Winterhuman** [comment]
Yep, set the Invidious API as the default in FreeTube, go into a browser and navigate to the same video (`6kolTgj7uQc`) on yewtu.be (or any Invidious instance), look at the replies under Philip Brechler's comments. https://yewtu.be displays "@ThePmso", FreeTube display `/channel/ID` in it's place.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Yep, set the Invidious API as the default in FreeTube, go into a browser and navigate to the same video (`6kolTgj7uQc`) on yewtu.be (or any Invidious instance), look at the replies under Philip Brechler's comment. https://yewtu.be displays "@ThePmso", FreeTube display `/channel/ID` in it's place.

im unable to replicate with the these steps

**Comment by Winterhuman** [comment]
Strange, what OS are you using? I'm not sure what other information I should give

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Strange, what OS are you using? I'm not sure what other information I should give

Win10

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Are u able to make a screenrecording on this. Im still unable to reproduce this.

**Comment by Winterhuman** [comment]
I think this is a Linux-only issue (and possibly MacOS? Would need someone else to test that). Last time I tried uploading a screenshot it didn't work so I don't think I'll be able to send a screenrecording unfortunately

**Comment by Winterhuman** [comment]
Nevermind, I've just realised something. https://yewtu.be correctly shows `@ThePmso`, but, https://invidious.snopyta.org (the Invidious instance FreeTube is configured with in my case) shows `/channel/UCl_bPpK-OdmlxSK_lhjItSQ`.

It's not an Invidious API problem, some Invidious instances just don't show `@user` links.
